src/features/common/Layout/Navbar/microComponents/NavbarMenuItem.tsx

```diff
@@ -0,0 +1,116 @@
+import type {
+  MenuItem,
+  MenuItemDescription,
+  MenuItemTitle,
+} from '../defaultTenantConfig';
+import type { JSX } from 'react';
+
+import { useMemo } from 'react';
+import Link from 'next/link';
+import { useLocale, useTranslations } from 'next-intl';
+import styles from '../NavbarMenu.module.scss';
+import {
+  PlatformIcon,
+  RedeemCodeIcon,
+  SupportIcon,
+  TransparencyIcon,
+  OrganizationIcon,
+  PartnerIcon,
+  TreeMapperIcon,
+  FireAlertIcon,
+  TracerIcon,
+  RestorationAdviceIcon,
+  RestorationStandardsIcon,
+  InstagramIcon,
+  LinkedinIcon,
+  YoutubeIcon,
+  FacebookIcon,
+  VTOChallengeIcon,
+  MangrovesChallengeIcon,
+} from '../../../../../../public/assets/images/icons/NavbarMenuIcons';
+import { addLocaleToUrl, isPlanetDomain } from '../utils';
+import { isAbsoluteUrl } from '../../../../../utils/apiRequests/apiClient';
+
+const navbarMenuIcons: Record<MenuItemTitle, JSX.Element> = {
+  platform: <PlatformIcon />,
+  redeemCode: <RedeemCodeIcon />,
+  support: <SupportIcon />,
+  organization: <OrganizationIcon />,
+  transparency: <TransparencyIcon />,
+  partner: <PartnerIcon />,
+  treeMapper: <TreeMapperIcon />,
+  fireAlert: <FireAlertIcon />,
+  tracer: <TracerIcon />,
+  restorationAdvice: <RestorationAdviceIcon />,
+  restorationStandards: <RestorationStandardsIcon />,
+  instagram: <InstagramIcon />,
+  youtube: <YoutubeIcon />,
+  linkedin: <LinkedinIcon />,
+  facebook: <FacebookIcon />,
+  vtoChallenge: <VTOChallengeIcon />,
+  mangroves: <MangrovesChallengeIcon />,
+};
+
+const excludedTitle = ['instagram', 'youtube', 'linkedin', 'facebook'];
+const isTranslatableTitle = (
+  title: string
+): title is Exclude<
+  MenuItemTitle,
+  'instagram' | 'youtube' | 'linkedin' | 'facebook'
+> => !excludedTitle.includes(title);
```

Comment:
Suggest using a type for the excluded titles for readability and clarity.

e.g.
```
type SocialMediaTitle = 'instagram' | 'youtube' | 'linkedin' | 'facebook';
const SOCIAL_MEDIA_TITLES: SocialMediaTitle[] = ['instagram', 'youtube', 'linkedin', 'facebook'];

const isTranslatableTitle = (
  title: MenuItemTitle
): title is Exclude<MenuItemTitle, SocialMediaTitle> => 
  !SOCIAL_MEDIA_TITLES.includes(title as SocialMediaTitle);
```

Reply:
This is irrelevant if the excluded titles are removed from the logic, but naming the type SocialMediaTitle makes the purpose clearer than ExcludedTitle in this case (as no other titles are excluded from translation)